### PR TITLE
AprilTag Updates

### PR DESCRIPTION
**Pull request by 122004**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/AprilTagDetection.java [hunk 1]
```diff
@@ -11,9 +11,9 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import java.util.ArrayList;
 
-public class AprilTagFinder extends SubsystemBase {
```
Comment (roboticsgeek) [inline src/main/java/frc/robot/subsystems/AprilTagDetection.java:14]: So I think this is mis-named. The subsystem is not a detection. The detection concept represents an individual instance/result of the detection process. The subsystem would be a "AprilTagDetector" It is the thing that creates and maintains detections, right?

**Review by roboticsgeek — COMMENTED** [review]
I think we want an AprilTagDetector not an AprilTagDetection which is already the wpilib output data type class. The detector does the detecting (from the point of view of the java code, even though it is just listening to the data stream from the video server).

File: src/main/java/frc/robot/subsystems/AprilTagDetection.java (hunk 1 above)
Comment (122004) [reply src/main/java/frc/robot/subsystems/AprilTagDetection.java:14]: Yeah that would make more sense. I'll close out this pull request and rename it at the meeting tonight.

**Comment by 122004** [comment]
Misspelling/confusion surrounding the WPILib output vs proper naming techniques for the subsystem itself. Completely my fault for misunderstanding, will fix at tonight's meeting.

**Pull request closed without merging**